src/qrcode/ScanQRScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import React, { useRef } from 'react'
+import { Dimensions } from 'react-native'
+import BackButton from 'src/components/BackButton'
+import i18n from 'src/i18n'
+import { emptyHeader } from 'src/navigator/Headers'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import QRScanner from 'src/qrcode/components/QRScanner'
+import { QrCode } from 'src/send/actions'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import Logger from 'src/utils/Logger'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.ScanQRScreen>
+
+export default function ScanQRScreen({ route, navigation }: Props) {
```

Comment:
this screen looks broken when camera permissions are denied 
<img src="https://github.com/valora-inc/wallet/assets/5062591/4fbaad2b-d053-4e7c-b75e-683ddd337c3f" width="250"/>